src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Border/Border_CornerRadius_BorderThickness.xaml.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples;
+using System.Linq;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+{
+	public sealed partial class BorderCornerRadiusBorderThickness : UserControl
+    {
+        public BorderCornerRadiusBorderThickness()
+        {
+            this.InitializeComponent();
+        }
+		public void Set_MyBorderThickness(Thickness t)
+		{
+			MyBorder.BorderThickness = t;
+		}
```

Comment:
you can actually get rid of this too; by making `MyBorder` public
and just set it directly, instead of using a helper method

Reply:
Make sense. I should have thought of that.